src/assets/less/main_window.less

```diff
@@ -194,9 +194,14 @@ html, body {
         width: 100%;
         font-size: 26px;
         line-height: 1.6em;
-        overflow: hidden; /* Hide scroll bar so that the user relies on auto scroll */
         padding: 12px 0;
       }
+      p[scroll=true] {
```

Comment:
It would make more sense to have an attribute represent a boolean value

I.e. The existance of `data-scroll` = true, if `data-scroll` is not set, then false.

In JS you would use `.setAttribute` and `.removeAttribute`